projects/media-viewer/src/lib/styles/media.scss

```diff
@@ -292,37 +292,37 @@
     background-color: hsla(0,0%,0%,.7);
   }
 
-  #outerContainer.sidebarOpen #viewerContainer {
+  sidebarOpen #viewerContainer {
```

Comment:
i believe you still need the dot ".sideOpen" in order to identify the class name

Reply:
fixed